.github/workflows/stylish.yml

```diff
@@ -5,34 +5,52 @@ on:
   workflow_dispatch:
 
 jobs:
-  stylish:
-    name: "black & flake8 & rpmlint"
+  check-spec:
+    name: rpmlint
     runs-on: ubuntu-latest
     container:
       image: fedora:latest
 
     steps:
-    - name: Base setup
-      run: |
-        dnf --setopt install_weak_deps=False install -y \
-            git-core \
-            python3-flake8 \
-            python3-pip \
-            rpmlint
-
-    - uses: actions/checkout@v3
-
-    - uses: psf/black@stable
-      with:
-        version: "22.3.0"
-
-    - name: Setup flake8 annotations
-      uses: rbialon/flake8-annotations@v1
```

Comment:
I'd still use the `rbialon/flake8-annotations@v1` action, so we get inline annotations for the issues